docs/quickstart.md

```diff
@@ -0,0 +1,54 @@
+# Quickstart
```

Comment:
We should also add how to setup the ingress domain stuff (i.e AName and CName records)